classification/class_task.py

```diff
@@ -165,7 +165,7 @@ def evaluate(self, test_loader):
             for data in tqdm(test_loader):
                 X, y = data
                 X, y = X.to(DEVICE), y.to(DEVICE)
-                preds = self.predict(X)
+                preds, _ = self.sample_predict(X)
```

Comment:
Possibly way too nit-picky, but should we instead change all of the sample_predict methods to just predict()? sample_predict doesn't really make sense for MLP

Reply:
done.